src/mumble_exe/CMakeLists.txt

```diff
@@ -54,10 +54,24 @@ if(packaging)
 		list(APPEND installer_vars "--all-languages")
 	endif()
 
+	# The installer uses different architecture naming and thus we have to convert at this point
+	set(ARCH "test")
+	if(MUMBLE_TARGET_ARCH STREQUAL "x64")
+		set(ARCH "x64")
+	elseif(MUMBLE_TARGET_ARCH STREQUAL "x86")
+		set(ARCH "x86")
+	else()
+		message(FATAL_ERROR "Unexpected target architecture ${MUMBLE_TARGET_ARCH}")
+	endif()
+
+	message(STATUS "Arch is set to ${ARCH}")
+
 	list(APPEND installer_vars
-		"--version" ${PROJECT_VERSION}
-		"--arch" ${ARCH}
+		"--version" "${PROJECT_VERSION}"
 	)
+#	list(APPEND installer_vars
+#		"--arch" "${ARCH}"
+#	)
```

Comment:
```suggestion
	list(APPEND installer_vars
		"--version" ${PROJECT_VERSION}
		"--arch" ${MUMBLE_TARGET_ARCH}
	)
```

Reply:
That won't work as the installer expects 64-bit to be labelled `x86_64` but `MUMBLE_TARGET_ARCH` will only be `x64`.

Plus I would really like to know what cmake's problem with the current code is. I just don't get it :eyes:

Reply:
What do you mean?

The architecture was originally explicitly passed, but `ARCH` is not set anymore because replaced by `MUMBLE_TARGET_ARCH`.

Also, the installer expects `x64`: https://github.com/mumble-voip/mumble/blob/20cd3350240976e27d6b17d3f5846537d3d0219d/installer/ClientInstaller.cs#L198-L200

Reply:
Ah indeed the values would even be compatible.

> but ARCH is not set anymore because replaced by MUMBLE_TARGET_ARCH.

That's not true. `ARCH` gets set right above before it gets used. Plus cmake also doesn't respect the --overlay and --g15 installer flags when actually invoking the installer. Something is profoundly wrong here...